include/sharg/detail/format_help.hpp

```diff
@@ -14,11 +14,16 @@
 
 #pragma once
 
-#include <seqan3/core/detail/test_accessor.hpp>
-
 #include <sharg/detail/format_base.hpp>
 #include <sharg/detail/terminal.hpp>
 
+namespace seqan3::detail
+{
+
+struct test_accessor;
+
+} // seqan3::detail
+
```

Comment:
This is a weird change. It is just as easy to copy over the respective file and put it into the sharg namespace, then replace the include. There is an issue for this #14 :)
Can you revert this change?

Reply:
Well, it removes the stuff that seqan3 includes in the `test_accessor` header, and I already adapted #14 to replace this forward declaration.

Btw, do we really need a header? because we do this declaration exactly once.